### ISSUE TITLE
How to reset settings to initial defaults?

### ISSUE THREAD

**Issue by ttodua** [dnSpyEx/dnSpy#116]
After you change settings, but then don't remember what you've changed, and just want to reset all settings/options to defaults, how to do that?
thanks in advance

**Comment by ElektroKill** [comment]
Hi, currently such functionality does not exist. The only way to reset the settings is to delete the config file in `%APPDATA%\dnSpy`. This wipes all the settings and restores everything to the default values. You can also open the file, find the relevant setting, and remove it. I might consider adding a button to reset the settings to default to the settings dialog. I'm not really sure of a use case for it though.

**Comment by ttodua** [comment]
Thanks for tip. Yes, it is very essential (basically, every software should have 'reset to defaults', as it's not software specific use-case, but it's like "about" button) because users change dozens of things/checkboxes from default and then don't remember (even if they rememeber, its not easy) how to revert them back.

**Comment by greenozon** [comment]
Absolutely, its very useful feature, taking into accout literally hundreds of options...
Just think - if a sw has a search box for options -> its super complicated :)
the idea is as highlighted above, eg:

![image](https://user-images.githubusercontent.com/765343/185735105-e567f068-c0b9-4a91-9f18-45c64f1851c9.png)

**Comment by ttodua** [comment]
I think it's quite trivial. We don't need complex "settings states" or etc at this moment. Let's start with simples - "Reset to defaults" which is does "delete the appdata/dnspy" folder. that's basic functionality, and if later anyone has more advanced needs, s/he might PR for that.
